common-test-utils/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.qe</groupId>
+        <artifactId>beefy-scenarios</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>common-test-utils</artifactId>
```

Comment:
This module breaks the independence of individual modules. Atm this helps to share one InfinispanTestResource in 2 modules.

I personally prefer to keep things loosely coupled with small degree of duplicates if needed.

@mjurc what's your view?

Reply:
@rsvoboda I agree

Reply:
Moving this discussion into the thread: https://github.com/quarkus-qe/beefy-scenarios/pull/40#discussion_r519770567